src/commands/lambda/prompt.ts

```diff
@@ -1,13 +1,6 @@
 import {blueBright, bold} from 'chalk'
-import {
-  CheckboxQuestion,
-  ConfirmQuestion,
-  InputQuestion,
-  ListQuestion,
-  prompt,
-  QuestionCollection,
-  Separator,
-} from 'inquirer'
+import {ConfirmQuestion, InputQuestion, ListQuestion, prompt, QuestionCollection, Separator} from 'inquirer'
+import inquirer from 'inquirer'
```

Comment:
I know these should be grouped into one import but for some reason I get an error when trying to import `inquirer` on line 2. https://a.cl.ly/7KuQkdkx
not really sure what is the issue with that

Reply:
hmmm I think I understand why I received the error, is it because I was sort of mixing the two import styles you described above?. The problem is that inquirer isn't some available property, instead what I did on line 3 was import the whole module and all its properties/methods as `inquirer` and that is no problem. Does this sound right?

I think to resolve this issue I should use the 2nd style of importing and prefix all the previously used inquirer stuff with `inquirer.`

Reply:
made the change in https://github.com/DataDog/datadog-ci/pull/513/commits/9fdd2b761d574c6020bfab0fb831d639df74e941

Reply:
Great answer, you're correct. You can prefix the rest of the methods with `inquirer`, or you could simply add the specific methods you used (registerPrompt, QuestionCollection, InputQuestion, ListQuestion, Separator, etc) to the existing import.